### ISSUE TITLE
elsa-workflow-instance-list-screen - Disable links to workflow-registry

### ISSUE THREAD

**Issue by noxe** [elsa-workflows/elsa-core#2512]
Hi,

i am using the elsa-workflow-instance-list-screen in our application - this list provides an hyperlink to filter by correlation id - but also has an link to the workflow-registry - which i dont use and also dont want to expose to the user - is there a way to disable the hyperlink?

    <elsa-studio-root server-url="@serverUrl" monaco-lib-path="_content/Elsa.Designer.Components.Web/monaco-editor/min">
        <elsa-workflow-instance-list-screen workflow-id="@this.Model.WorkflowDefinitionId" />
    </elsa-studio-root>

**Comment by sfmskywalker** [comment]
Indeed, we need to allow the component user to provide a path template.

**Comment by noxe** [comment]
should not be stale :)
